### PR TITLE
Add default organization to onboarding flow

### LINKED ISSUE #3970: "Demo Lab" should be selected by default for onboarding
On wk.org, the "Demo Lab" is the only organization with auto activation. Hence, we want people to join this org. However, when signing up through the `onboarding flow` the "demo lab" is not selected by default. 

image

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
Added a default organization that is used in the onboarding flow. The organization is set in `application.conf -> features`. 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Log out and open `<base_url>/onboarding`.
- Now you can select an organization. The default organization should be the one that is defined in `application.conf`.

### Issues:
- fixes #3970

------
- ~~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -92,6 +92,8 @@ features {
   # if enableFrontpage is true, / will always be the /spotlight page
   # if it is false, / will be the login page, if not logged in, and the dashboard otherwise
   enableFrontpage = false
+  # The default organization used in the onboard flow (wrapped in "). If you don't want one use null or "".
+  defaultOnboardingOrganization = null
```
Comment (philippotto) [inline conf/application.conf]: There already is a `defaultOrganization` within the application.conf. Please use this attribute instead. Unless there is a specific reason against it?

Comment (MichaelBuessemeyer) [reply conf/application.conf]: The reason I did not use it in the first iteration is, that there might be the case where you want the default organization to be different in onboarding and in the usual registration (afaik the current application of `defaultOrganization`).
If you think they both are related and should be always the same, I will change it :slightly_smiling_face: I was just unsure about this :)

Comment (philippotto) [reply conf/application.conf]: The relevant use case for us is the demo deployment for which we always want to default to the demo org. I cannot think of a scenario, in which you'd want two different defaults depending on which flow you used :)

**Comment by MichaelBuessemeyer** [comment]
@philippotto In the last commit, I added the fix from #3419 as a script. This just kept bothering me a bit, as I always have to execute this specific command after each time I refresh the snapshots.
I added the script to `refresh-snapshots` by default.
If the problem does not occour for you two (@philippotto  @daniel-wer), I will remove this and suggest to keep this as an optional script for me.

Otherwise, I have to always open fm3's solution and execute it manually :)

**Comment by philippotto** [comment]
> @philippotto In the last commit, I added the fix from #3419 as a script. This just kept bothering me a bit, as I always have to execute this specific command after each time I refresh the snapshots.
> I added the script to `refresh-snapshots` by default.
> If the problem does not occour for you two (@philippotto @daniel-wer), I will remove this and suggest to keep this as an optional script for me.

I have to execute this from time to time, as well, but it's not coupled to `refresh-snapshots` for me. I'd recommend to put the command into a `sh` file in the wk repo (does not need to be checked in). Then, you can easily execute this. Personally, I just find & execute the command via bash's `ctrl+r` search.

**Comment by philippotto** [comment]
> > @philippotto In the last commit, I added the fix from #3419 as a script. This just kept bothering me a bit, as I always have to execute this specific command after each time I refresh the snapshots.
> > I added the script to `refresh-snapshots` by default.
> > If the problem does not occour for you two (@philippotto @daniel-wer), I will remove this and suggest to keep this as an optional script for me.
> 
> I have to execute this from time to time, as well, but it's not coupled to `refresh-snapshots` for me. I'd recommend to put the command into a `sh` file in the wk repo (does not need to be checked in). Then, you can easily execute this. Personally, I just find & execute the command via bash's `ctrl+r` search.

#3419 was closed today :tada: So, this shouldn't be necessary, anymore!

**Comment by MichaelBuessemeyer** [comment]
@philippotto I removed the script in package.json now. Could you please review/approve these changes :)?

**Comment by jstriebel** [comment]
@MichaelBuessemeyer Is the `defaultOrganization` a new config option? If yes please add it to the migration guide (and maybe also the changelog). If it was used before already nvm :-)

**Comment by MichaelBuessemeyer** [comment]
@jstriebel It is not new, it was used before :slightly_smiling_face: .

**Pull request merged**